backend/cmd/headlamp.go

```diff
@@ -245,6 +245,24 @@ func defaultKubeConfigPersistenceFile() (string, error) {
 // It serves plugin static files at “/plugins/” and “/static-plugins/”.
 // It disables caching and reloads plugin list base paths if not in-cluster.
 func addPluginRoutes(config *HeadlampConfig, r *mux.Router) {
+	// Delete plugin route
+	// This is only available when running locally.
+	if !config.useInCluster {
```

Comment:
This should be protected somehow.

Reply:
Sorry I didn't get you, there is already a check for Headlamp backend token. Is there any other token that I am missing here.